### PR TITLE
Happo integration

### PR DESCRIPTION
**Pull request by sslotsky**
## Reason for change

Set up happo.io for visual regression testing.

### CODE REVIEW

File: src/components/manifold-product-page/manifold-product-page.e2e.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { newE2EPage } from '@stencil/core/testing';
-import { Product, Provider } from '../../spec/mock/catalog';
```
Comment (sslotsky) [inline src/components/manifold-product-page/manifold-product-page.e2e.ts]: This was failing TS compilation by the way, not sure how it made it in :confused:

Comment (davidleger95) [reply src/components/manifold-product-page/manifold-product-page.e2e.ts]: I don't think it will fail for test files

**Comment by davidleger95** [comment]
Should I be reviewing these PRs, or is this just a Platform Squad concern? Not really sure what's going on with Happo stuff, so my review won't mean much. 🤷‍♂ I'll click the button if that's all you need though 😄

**Comment by sslotsky** [comment]
@davidleger95 I'd definitely like for DX to be reviewing this. I think it will be up to DX devs to write more visual regression tests, so looking this over for any concerns or questions you might have is important.

**Review by davejsdev — APPROVED** [review]
looks good to me!

**Pull request merged**